glyphs-reader/src/font.rs

```diff
@@ -1862,6 +1885,31 @@ impl RawFeature {
     fn disabled(&self) -> bool {
         self.disabled == Some(1)
     }
+
+    // https://github.com/googlefonts/glyphsLib/blob/24b4d340e4c82948ba121dcfe563c1450a8e69c9/Lib/glyphsLib/builder/features.py#L134
+    fn feature_names(&self) -> String {
+        if self.labels.is_empty() {
+            String::new()
+        } else {
+            let labels = self
+                .labels
+                .iter()
+                .map(|label| {
+                    let language_id = GLYPHS_TO_OPENTYPE_LANGUAGE_ID
+                        .iter()
+                        .find(|entry| entry.0 == label.language)
```

Comment:
not critical since this is not a hot code path, but in general I'd prefer doing a binary search to a linear scan here; the one gotcha is we would want to ensure that the array is sorted (which we could do with a test case that created an explicitly sorted copy and compared this with the original)